core/storage/ipfs/datastore.hpp

```diff
@@ -51,8 +53,39 @@ namespace fc::storage::ipfs {
      * @return success if removed or didn't exist, error otherwise
      */
     virtual outcome::result<void> remove(const CID &key) = 0;
-  };
 
+    /** Compute CID from bytes */
```

Comment:
Triple slash can be used for one-line docstrings
```suggestion
    /// Compute CID from bytes
```